examples/SoundClassification/sketch.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019 ml5
```

Comment:
If existing examples written using p5.js work similarly well to this (vanilla-JS) one, I'd suggest using the p5.js-based ones (without async/await) instead.